src/python/tests_extended/test_export_to_onnx.py

```diff
@@ -264,71 +265,76 @@
 }
 
 EXPECTED_RESULTS = {
-    'AveragedPerceptronBinaryClassifier': {'cols': [('PredictedLabel', 'PredictedLabel.0')]},
+    'AveragedPerceptronBinaryClassifier': {'cols': [('PredictedLabel', 'PredictedLabel')]},
     'CharTokenizer': {'cols': [('SentimentText_Transform.%03d' % i, 'SentimentText_Transform.%03d' % i)
                       for i in range(0, 422)]},
-    'ColumnDuplicator': {'cols': [('dup', 'dup.0')]},
+    'ColumnConcatenator': {'num_cols': 11, 'cols': 0},
+    'ColumnDuplicator': {'cols': [('dup', 'dup')]},
     'ColumnSelector': {
         'num_cols': 2,
-        'cols': [('Sepal_Width', 'Sepal_Width.0'), ('Sepal_Length', 'Sepal_Length.0')]
+        'cols': [('Sepal_Width', 'Sepal_Width'), ('Sepal_Length', 'Sepal_Length')]
     },
-    #'EnsembleClassifier': {'cols': [('PredictedLabel', 'PredictedLabel.0')]},
-    #'EnsembleRegressor': {'cols': [('Score', 'Score.0')]},
-    'FastForestBinaryClassifier': {'cols': [('PredictedLabel', 'PredictedLabel.0')]},
-    'FastForestRegressor': {'cols': [('Score', 'Score.0')]},
-    'FastLinearBinaryClassifier': {'cols': [('PredictedLabel', 'PredictedLabel.0')]},
-    'FastLinearClassifier':  {'cols': [('PredictedLabel', 'PredictedLabel.0')]},
-    'FastLinearRegressor': {'cols': [('Score', 'Score.0')]},
-    'FastTreesBinaryClassifier':  {'cols': [('PredictedLabel', 'PredictedLabel.0')]},
-    'FastTreesRegressor': {'cols': [('Score', 'Score.0')]},
-    'FastTreesTweedieRegressor': {'cols': [('Score', 'Score.0')]},
-    'FromKey': {'cols': [('Sepal_Length', 'Sepal_Length.0'), ('Label', 'Label.0')]},
+    #'EnsembleClassifier': {'cols': [('PredictedLabel', 'PredictedLabel')]},
+    #'EnsembleRegressor': {'cols': [('Score', 'Score')]},
+    'FastForestBinaryClassifier': {'cols': [('PredictedLabel', 'PredictedLabel')]},
+    'FastForestRegressor': {'cols': [('Score', 'Score')]},
+    'FastLinearBinaryClassifier': {'cols': [('PredictedLabel', 'PredictedLabel')]},
+    'FastLinearClassifier':  {'cols': [('PredictedLabel', 'PredictedLabel')]},
+    'FastLinearRegressor': {'cols': [('Score', 'Score')]},
+    'FastTreesBinaryClassifier':  {'cols': [('PredictedLabel', 'PredictedLabel')]},
+    'FastTreesRegressor': {'cols': [('Score', 'Score')]},
+    'FastTreesTweedieRegressor': {'cols': [('Score', 'Score')]},
+    'FromKey': {'cols': [('Sepal_Length', 'Sepal_Length'), ('Label', 'Label')]},
     'GlobalContrastRowScaler': {'cols': [
-        ('normed_columns.Petal_Length', 'normed_columns.0'),
-        ('normed_columns.Sepal_Width', 'normed_columns.1'),
-        ('normed_columns.Sepal_Length', 'normed_columns.2')
+        ('normed_columns.Petal_Length', 'normed_columns.Petal_Length'),
+        ('normed_columns.Sepal_Width', 'normed_columns.Sepal_Width'),
+        ('normed_columns.Sepal_Length', 'normed_columns.Sepal_Length')
     ]},
```

Comment:
Does this fix work for columns with multiple slots? Also see the other ones below which also contain more than one slot.

Reply:
no, the update here is due to Harish fix for slot names.

---
In reply to: [382866405](https://github.com/microsoft/NimbusML/pull/445#discussion_r382866405) [](ancestors = 382866405)